app/views/page/show.html.erb

```diff
@@ -174,8 +174,23 @@
               <i class="far fa-file margin-right-1"></i><%= display_name != '' ? display_name : component.attachment_file_name %>
             <% end %>
           </div>
-
-        <%# Text and Images %>
+          <%# Map Component %>
+        <% when 'PageMapComponent' %>
+          <%
+            description = component.description
+            title = component.title
+            next_component = @page.page_components.find_by(position: pc.position + 1)
+          %>
+          <div class="margin-bottom-2 font-sans-lg text-bold margin-top-9">
+            <%= content_tag(:span, title, class: 'margin-right-1') if title != '' %>
+          </div>
+          <div class="margin-bottom-1 page-map-component">
+            <%= render partial: "page/page_map", locals: { component: component } %>
+          </div>
+          <div class="margin-top-2 grid-row <%= page_narrow_classes %> <%= next_component != nil && next_component.component_type == 'margin-bottom-9 ' %> font-sans-sm line-height-162<%= ' margin-bottom-9' if last_component === index %>">
+            <%= content_tag(:span, description, class: 'margin-right-1') if description != '' %>
+          </div>
+            <%# Text and Images %>
```

Comment:
Can we put a space above the comment and align it with the `<% when %>`?

Reply:
Got it.